packages/oauth/src/index.ts

```diff
@@ -221,25 +227,28 @@ export class InstallProvider {
             }
 
             // Update Installation
-            installationUpdates[tokenType].token = refreshResp.access_token;
-            installationUpdates[tokenType].refreshToken = refreshResp.refresh_token;
-            installationUpdates[tokenType].expiresAt = currentUTCSec + refreshResp.expires_in;
-
-            const updatedInstallation = {
-              ...installationUpdates,
-              [tokenType]: { ...queryResult[tokenType], ...installationUpdates[tokenType] },
-            };
-
-            // TODO: related to the above TODO comment as well
-            // eslint-disable-next-line no-await-in-loop
-            await this.installationStore.storeInstallation(updatedInstallation);
```

Comment:
If both bot and user tokens are refreshed, the current code saves those refreshed token separately (twice in the loop at maximum). However, this can be done at a time. @misscoded Let me know if you see any issues with this.